src/plugins/paginate/_base.scss

```
@@ -24,6 +44,79 @@
 		border-bottom-right-radius: 4px;
 		border-top-right-radius: 4px;
 	}
+
+	li.active:nth-last-child(2) button {
+		border-bottom-right-radius: 4px;
+		border-top-right-radius: 4px;
+	}
+
+	.pagination {
+		& > li {
+			&.active {
+				& > button {
+					cursor: default;
+					outline-offset: -2px;
+				}
+			}
+			& > button {
+				background-color: #eaebed;
+				border: 1px solid #dcdee1;
+				color: #335075;
+				cursor: pointer;
+				display: inline-block;
+				float: left;
+				line-height: 1.4375;
+				margin-bottom: .5em;
+				margin-left: -1px;
+				padding: 10px 16px;
+				position: relative;
+				text-decoration: none;
+
+				&:hover {
+					@include hover-focus;
+				}
+				&:focus {
+					@include hover-focus;
+				}
+			}
```

This can be optimized (see suggestion). Also, the variables are already available in "`src/_variables.scss`". I have updated the code to use them.

```suggestion
			& > button {
				background-color: $btn-default-bg;
				border: 1px solid $btn-default-border;
				color: $btn-default-color;
				margin-bottom: .5em;
				margin-left: -1px;
				padding: 10px 16px;
				position: relative;

				&:hover, &:focus {
					background-color: #d4d6da;
					border-color: #bbbfc5;
					z-index: 2;
				}
			}
```